content/01.abstract.md

```diff
@@ -1,20 +1,11 @@
 ## Abstract {.page_break_before}
 
-Deep learning, which describes a class of machine learning algorithms, has
```

Comment:
Should we try to merge some of the outstanding pull requests first?  That may take too long, so I'm okay updating the line wrapping now if you think that is most efficient.  We will need to provide help with merge conflicts for those pull requests though.

Reply:
I only reflowed the abstract assuming that none of the PRs touch the abstract. Yes, we should focus on reviewing exiting PRs. They likely all have conflicts now. Tag me when it's time to merge and I will handle the conflict resolution. Sooner the better.

Reply:
@dhimmel ah, I see.  I reviewed this hastily between meetings.